reporting/tests/Dockerfile

```diff
@@ -7,5 +7,4 @@ RUN pip install -r requirements.txt
 COPY scenarios scenarios/
 COPY tests tests/
 
-# CMD ["bash", "tests/run_tests.sh" , "&&", "while true; do sleep 1; done"]
 CMD [ "bash", "tests/run_tests.sh" ]
```

Comment:
If you change this to `ENTRYPOINT` then from the CLI the syntax can be the same:

```
docker compose run --build tests "SIDARTHE Equations" "SIR"
```

etc

Reply:
Ah nice, I will update that since it's cleaner.